runtime/queries/metricsview_timeseries_test.go

```diff
@@ -33,32 +33,20 @@ func TestMetricsViewsTimeseries_month_grain(t *testing.T) {
 
 	err = q.Resolve(context.Background(), rt, instanceID, 0)
 	require.NoError(t, err)
-	require.NotEmpty(t, q.Result)
-	rows := q.Result.Data
-	i := 0
-	require.Equal(t, parseTime(t, "2023-01-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-02-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-03-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-04-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-05-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-06-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-07-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-08-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-09-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-10-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-11-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
-	i++
-	require.Equal(t, parseTime(t, "2023-12-01T00:00:00Z").AsTime(), rows[i].Ts.AsTime())
+	assertTimeSeriesResponse(t, q.Result, []string{
```

Comment:
I think it will be slightly harder to find a timestamp that differs, the error line now always points to the first line.

Reply:
Ah that was the reason to do something like this. Perhaps it would be better to convert both to strings and compare lists?